contracts/eos/BancorConverter/migrate.cpp

```diff
@@ -0,0 +1,116 @@
+[[eosio::action]]
+void BancorConverter::migrate( const set<symbol_code> converters )
```

Comment:
wasn't aware you can create multiple cpp files for a single contract

Reply:
🤓 yep, makes life a lot easier by segmenting the logic into different files.